pyproject.toml

```diff
@@ -36,7 +36,6 @@ sqlalchemy = [
 sqlalchemy-with-utils = [
     "Flask-Admin[sqlalchemy]",
     "sqlalchemy_utils>=0.38.0",
-    "sqlalchemy-citext>=1.8.0",
```

Comment:
I've had a bit more of a think about this since my last comment - sorry.

We still theoretically support `sqlalchemy<2.0.19` - this feels like it would break users on SQLAlchemy v1 and I'm not sure we've made the decision to drop support for that yet.